tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -18,25 +18,20 @@ use base "y2x11test";
 use strict;
 use warnings;
 use testapi;
-use utils 'clear_console';
-use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
+use utils qw(type_string_slow_extended clear_console);
 
 sub run {
     my $self         = shift;
     my $module       = "host";
-    my $dm           = lc get_var('DESKTOP');
+    my $dm           = lc get_required_var('DESKTOP');
     my $hosts_params = {
         ip    => '195.135.221.134',
         fqdn  => 'download.opensuse.org',
         alias => 'download-srv'
     };
-    my %execute = (
-        gnome => \&turn_off_gnome_screensaver,
-        kde   => \&turn_off_kde_screensaver
-    );
-    $execute{$dm}->();
```

Comment:
Is this change on purpose? If yes, we can remove `$dm` initialization too.

Reply:
yup, thanks!